irohad/main/irohad.cpp

```diff
@@ -139,6 +145,32 @@ int main(int argc, char *argv[]) {
       return EXIT_FAILURE;
     }
 
+    // Check if force flag to overwrite genesis block specified
+    if (not FLAGS_overwrite_genesis) {
+        auto genesis_block_exist = false;
+        irohad.storage->getBlockQuery()->getTopBlocks(1).subscribe(
+            [&genesis_block_exist](auto block) { genesis_block_exist = true; });
+
+        // check if genesis block already existing
+        if (genesis_block_exist) {
+            std::string choice;
+            std::cout << "Warning : " << config[mbr::BlockStorePath].GetString()
```

Comment:
There should not be any interactive part, like `cout` or `getline`. It should work similarly to `git push --force`.
So if we pass genesis block: 
- if flag is not set and there are blocks in ledger, we output a message in `log_` and exit.
- if flag is not set and there are no blocks in ledger, we continue normally
- if flag is set and there are blocks in ledger, we output a message in `log_` that we are overwriting the ledger, and continue normally
- if flag is set and there are no blocks in ledger, we continue normally